### DISCUSSION TITLE
Share your Smart Powermeter setup!

### DISCUSSION THREAD

**Discussion by JGAguado** [JGAguado/Smart_Powermeter#19] [Show and tell]
I would love to know more about the ⚡Smart Powermeter ⚡ around the world!

How does it look your 🔌 setup 🔌? Are you using it for very specific 🧰 applications 🧰? Did you get to gain better insights of your electric grid 💰?

Use this discussion to share some pictures and/or config files and let us know about fresh uses of your ⚡Smart Powermeter ⚡!

![image](https://github.com/JGAguado/Smart_Powermeter/assets/7847822/588c48df-d35b-4580-8ae6-e47e7e0ac30a)

**Comment by AdamJHowell** [comment]
Here's my haphazard setup:
![20240514_232016](https://github.com/JGAguado/Smart_Powermeter/assets/698228/2e7d7003-76c5-4dca-b62e-1a7411ee2f23)
I used a "sharpie" to mark each clamp with its Smart-PowerMeter port number.  1 and 2 are the mains input (US split-phase 240).  3 and 4 are 120v rooms.  5 and 6 are the two 120v phases of the 240v air conditioner.

![20240514_232611](https://github.com/JGAguado/Smart_Powermeter/assets/698228/2ac78e6a-0baa-4022-80ab-6b415ad04d5f)
I still do not have the device mounted properly.  I opted to use it outside of the breaker box.  US panels have dozens of pre-punched holes that can be removed without tools, so I used one of those.  My plan is to mount it right next to the breaker box.

My Home Assistant setup is very stock right now.  Next I will create power entities for the individual inputs so I can track the power used by the A/C and the two rooms that I'm monitoring.  Once all of that is done, I'll probably buy another so I can track more rooms.

**Reply by AdamJHowell** [reply]
Oh, I also put small zip-ties on each CT clamp wire near the plug.  Clamp #1 has one zip tie, #2 has two zip ties, and so on.  This makes it easy to get the correct clamp plugged into the correct port.  This is really helpful when the unit is installed outside of the panel.

**Reply by JGAguado** [reply]
Very cool @AdamJHowell ! Looking forward more insights you get after finishing the HA setup! 😃

**Comment by lucazeta1975** [comment]
![IMG_20240921_165729](https://github.com/user-attachments/assets/f6599bf4-9d4c-4df9-95a1-38c6bb9fd853)

![IMG_20240921_165744](https://github.com/user-attachments/assets/309d482b-fb8a-46a5-84aa-a3f90f8b3eb5)

![IMG_20240921_165815](https://github.com/user-attachments/assets/dc4a42ca-3337-4b87-a5cf-dc39a7f57e50)

I just moved to a different font LOL

**Reply by JGAguado** [reply]
Very cool 😎😎!
